content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,317 @@
 ---
 layout: "enterprise"
-page_title: "Logging - Infrastructure Administration - Terraform Enterprise"
+page_title: "Log Forwarding - Infrastructure Administration - Terraform Enterprise"
 ---
 
-# Terraform Enterprise Logs
+# Terraform Enterprise Log Forwarding
 
-This document contains information about interacting with Terraform Enterprise logs. There are two types of logs, application logs and audit logs. Application logs emit information about the services that comprise Terraform Enterprise. Audit logs emit information whenever any resource managed by Terraform Enterprise is changed.
+Terraform Enterprise supports forwarding its logs to one or more external
+destinations, a process called log forwarding. Log forwarding provides increased
+observability, assistance complying with log retention requirements, and
+information during troubleshooting.
 
-## Application Logs
+## Requirements
 
-Terraform Enterprise runs in a set of Docker containers. As such, any tooling that can interact with Docker logs can read the logs. This includes the command `docker logs`, as well as access via the [Docker API](https://docs.docker.com/engine/api/v1.36/#operation/ContainerLogs).
+Log forwarding requires:
 
-An example of a tool that can automatically pull logs for all docker containers is [logspout](https://github.com/gliderlabs/logspout).
-Logspout can be configured to take the Docker logs and send them to a syslog endpoint. Here's an example invocation:
+- Terraform Enterprise running on an instance using `systemd-journald`. Execute
+`systemctl status systemd-journald` to check if the `systemd-journald` service
+is started and enabled.
+- A version of Docker that supports the `journald` logging driver. Execute
```

Comment:
I feel the "which version, though?" question could be mitigated if we can add the specific version it started in. Is that a clear thing to add?

Reply:
Based on a quick parsing of [Docker's changelog](https://github.com/moby/moby/blob/master/CHANGELOG.md), I see the earliest version of Docker to mention the `journald` logging driver is 1.9.0. However, Terraform Enterprise requires a minimum Docker version of 17.06 for the community edition and 1.13 for the version shipped in RHEL extras. That's why I figured running a command to verify the plugin is listed would be better than giving a version that may or may not work.